packages/itmat-interface/test/serverTests/study.test.ts

```diff
@@ -1038,8 +1050,10 @@ describe('STUDY API', () => {
                         id: `user_access_obj_user_id_${createdUserAuthorisedStudy.id}`,
                         projects: [],
                         studies: []
-                    }
+                    },
                 });
+                expect((res.body.data.createUser.expiredAt - 
+                    res.body.data.createUser.createdAt) / 1000).toBeCloseTo(86400.0, 1);
```

Comment:
Is there a reason not to keep the comparison in milliseconds ?

Reply:
Cause I found when testing on MacOS, the error canbe 1ms sometimes, which is on the digit instead of the decimal part.